identity/backend/src/main/resources/application.yaml

```diff
@@ -1,13 +1,7 @@
 spring:
-  web:
-    resources:
-      cache:
-        cachecontrol:
-          max-age: 31536000
-          s-max-age: 31536000
-          must-revalidate: true
-          proxy-revalidate: true
-          no-transform: true
+  datasource:
```

Comment:
@Ben-Sheppard, in the beginnings of the single application, we learned to have the `application` property files in a central place, otherwise, we run into issues that the application properties where loaded in a different order or some weren't applied at all, etc.

That's why, could you please move this configuration to the `application.properties` file in the `dist` module. That way, we have that in a central place, and do not run into issue in which order they are loaded.